demosplan/DemosPlanCoreBundle/Logic/Report/ReportService.php

```diff
@@ -347,4 +341,15 @@ protected function addAnonymizationReport(string $category, StatementAnonymizeRp
 
         return $this->reportRepository->addObject($report);
     }
+
+    public function persistReportEntry(array $reportEntries, EntityManagerInterface $em): void
```

Comment:
should be named `persistReportEntries` when an array is passed. Why do you inject the entity manager? Isn't it possible to use DI in the ReportService?

Reply:
done in https://github.com/demos-europe/demosplan-core/pull/3670/commits/b73a623b3a91c6f4af9793abb24e3245455f28c0